### PR TITLE
Clone pause actions

### PR DESCRIPTION
**Pull request by Dinaamagdy**

### CODE REVIEW

File: tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py [hunk 1]
```diff
@@ -223,6 +223,85 @@ def test002_start_stop_vm(self):
...
+        self.log("Pause [VM1], should succceed..")
+        temp_actions = {'node': {'actions': ['pause'], 'service': self.vm1}}
+        res = self.create_vm(accounts=self.accounts, cloudspaces=self.cloudspaces,
+                             vms=self.vms, temp_actions=temp_actions)
+        self.wait_for_service_action_status(self.vm1, res[self.vm1]['pause'])
+
+        self.log("Check that [VM1] is PAUSED.")
+        cloudspaceId = self.get_cloudspace(self.cs1)['id']
+        vm = self.get_vm(cloudspaceId, self.vm1)
+        self.assertEqual(vm["status"], "HALTED")
```
Comment (john-kheir) [inline tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py]: Paused not halted

Comment (Dinaamagdy) [reply tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py]: done

File: tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py [hunk 2]
```diff
@@ -223,6 +223,85 @@ def test002_start_stop_vm(self):
...
+    def test004_clone_vm(self):
+        """ ZRT-OVC-014
+        *Test case for testing clone vm .*
+
+        **Test Scenario:**
+
+        #. Create a vm[vm1], should succeed.
+        #. Stop [VM1], should succeed.
+        #. Clone VM1 as [VM2_C], should succeed.
+        #. Check that the cloned vm[VM2_C] has been created and got ipaddreess.
```
Comment (john-kheir) [inline tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py]: --> ip address

Comment (Dinaamagdy) [reply tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py]: done

File: tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py [hunk 3]
```diff
@@ -223,6 +223,85 @@ def test002_start_stop_vm(self):
...
+
+        self.log("Stop [VM1], should succeed.")
+        temp_actions = {'node': {'actions': ['stop'], 'service': self.vm1}}
+        res = self.create_vm(accounts=self.accounts, cloudspaces=self.cloudspaces,
+                             vms=self.vms, temp_actions=temp_actions)
+        self.wait_for_service_action_status(self.vm1, res[self.vm1]['stop'])
+
+        self.log(" Clone VM1 as [VM2_C], should succeed.")
+        vm2_c = self.random_string()
+        temp_actions = {'node': {'actions': ['stop'], 'service': self.vm1,
```
Comment (john-kheir) [inline tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py]: --> clone

Comment (Dinaamagdy) [reply tests/integration_tests/testsuite/ovc_tests/a_basic/vms_tests.py]: done

**Pull request merged**